app/helpers/application_helper.rb

```diff
@@ -278,4 +278,8 @@ def sign_up_path
       new_user_registration_path
     end
   end
+
+  def unread_notifications(user)
+    @unread_notifications = Comment.find_since_last_login(user)
```

Comment:
Is it necessary to assign it to an instance variable instead of just return it?
And is this method necessary at all?

@hennevogel what do you think?